lib/compiler.js

```diff
@@ -85,6 +85,10 @@ module.exports = function compile(options) {
   var modelInstructions = buildAllModelInstructions(
     modelsRootDir, modelsConfig, modelSources);
 
+  var mixinDirs = options.mixinDirs || modelsMeta.mixins || ['./mixins'];
```

Comment:
`mixinDirs` should be empty by default. The value `['./mixins']` should be used as default for `mixinSources` only.